src/base/handlers.py

```diff
@@ -87,6 +89,12 @@ def _GetXsrfKey():
   return xsrf_key
 
 
+def GetCspNonce():
```

Comment:
Maybe _GetCspNonce()?  I don't think we expect anyone outside of here to need to call this, right?